wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
+import socket
+
+import requests
+from requests import Session
+import urllib3
+
+class wesClient:
+    def __init__(self,url:str, base_path:str, token: Optional[str] = None,)-> None:
+        """Class Constructor"""
+        self.url = url
+        self.base_path = base_path
+        self.headers = {"Accept": "application/json"}
+        self.session = Session()
+        self.session.headers.update(self.headers)  
+        self.token = token
+        
+    def get_runs(self,token:Optional[str] = None):
+        url = f"{self.url}{self.base_path}/runs"
+        response = self.session.get(url)
+        if token:
+            self.token = token
+            self.headers
+        try:
+            response = self.session.get(url)
+            data = response.json()
+        except (
```

Comment:
Better to use the following pattern to keep the info/trace from the original exception (see [docs](https://docs.python.org/3/tutorial/errors.html#exception-chaining)):

```py
except (MyException1, MyException2, MyException3) as exc:
    raise MyOneException("My one message") from exc
```

Also, you don't expect these exceptions to happen on the `data = response.json()` call, so it's best to put that line outside the `try` - `except` block.

In fact you might as well simply do `return response.json()`.